package.json

```diff
@@ -65,13 +65,12 @@
     "node-polyglot": "^2.2.2",
     "npm-run-all": "^4.0.1",
     "postcss-loader": "^2.0.6",
-    "preact": "^8.1.0",
-    "preact-compat": "^3.13.1",
+    "prettier": "^1.12.1",
```

Comment:
Pourquoi avoir ajouté `prettier` ? C'est normalement `eslint-config-cozy-app` qui s'occupe de faire l'import de prettier.

Reply:
That is a side-effect from the rebase. I will remove it and use cozy-scripts configuration files.